Tests/Tests/STPBankAccountParamsTest.m

```diff
@@ -0,0 +1,100 @@
+//
+//  STPBankAccountParamsTest.m
+//  Stripe
+//
+//  Created by Joey Dong on 6/19/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+@import XCTest;
+
+#import "STPBankAccountParams.h"
+
+@interface STPBankAccountParams ()
+
+- (NSString *)accountHolderTypeString;
+
+@end
+
+@interface STPBankAccountParamsTest : XCTestCase
+
+@end
+
+@implementation STPBankAccountParamsTest
+
+- (void)setUp {
+    [super setUp];
+    // Put setup code here. This method is called before the invocation of each test method in the class.
+}
+
+- (void)tearDown {
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+    [super tearDown];
+}
+
+#pragma mark -
+
+- (void)testLast4ReturnsAccountNumberLast4 {
+    STPBankAccountParams *bankAccountParams = [[STPBankAccountParams alloc] init];
+    bankAccountParams.accountNumber = @"000123456789";
+    XCTAssertEqualObjects(bankAccountParams.last4, @"6789");
+}
+
+- (void)testLast4ReturnsNilWhenNoAccountNumberSet {
+    STPBankAccountParams *bankAccountParams = [[STPBankAccountParams alloc] init];
+    XCTAssertNil(bankAccountParams.last4);
+}
+
+- (void)testLast4ReturnsNilWhenAccountNumberIsLessThanLength4 {
+    STPBankAccountParams *bankAccountParams = [[STPBankAccountParams alloc] init];
+    bankAccountParams.accountNumber = @"123";
+    XCTAssertNil(bankAccountParams.last4);
+}
+
+- (void)testAccountHolderTypeString {
+    STPBankAccountParams *bankAccountParams = [[STPBankAccountParams alloc] init];
+
+    bankAccountParams.accountHolderType = STPBankAccountHolderTypeIndividual;
+    XCTAssertEqualObjects([bankAccountParams accountHolderTypeString], @"individual");
+
+    bankAccountParams.accountHolderType = STPBankAccountHolderTypeCompany;
+    XCTAssertEqualObjects([bankAccountParams accountHolderTypeString], @"company");
+}
+
+#pragma mark - Description Tests
+
+- (void)testDescriptionWorks {
+    STPBankAccountParams *bankAccountParams = [[STPBankAccountParams alloc] init];
+    bankAccountParams.accountNumber = @"000123456789";
+    bankAccountParams.routingNumber = @"123456789";
+    bankAccountParams.country = @"US";
+    bankAccountParams.currency = @"usd";
+    bankAccountParams.accountHolderName = @"John Doe";
+    bankAccountParams.accountHolderType = STPBankAccountHolderTypeCompany;
+    XCTAssert(bankAccountParams.description);
```

Comment:
nit: should we use a more strict assertion here? I think this would still pass if we didn't implement a custom description. Maybe assert that the description contains the class name?